demo/app/components/demo-map/demo-map.component.ts

```diff
@@ -1,46 +1,85 @@
-import { SceneMode } from '../../../../src/angular-cesium/models/scene-mode.enum';
 
 @Component({
-	selector : 'demo-map',
-	templateUrl : './demo-map.component.html',
-	providers : [ViewerConfiguration],
-	styleUrls : ['./demo-map.component.css']
+  selector: 'demo-map',
+  templateUrl: './demo-map.component.html',
+  providers: [ViewerConfiguration],
+  styleUrls: ['./demo-map.component.css'],
 })
-export class DemoMapComponent {
-	@ViewChild('layer') tracksLayer: TracksLayerComponent;
-	arcGisMapServerProvider = MapLayerProviderOptions.ArcGisMapServer;
-	sceneMode = SceneMode.SCENE3D;
-	
-	constructor(private viewerConf: ViewerConfiguration,
-							private appSettingsService: AppSettingsService) {
-		viewerConf.viewerOptions = {
-			selectionIndicator : false,
-			timeline : false,
-			infoBox : false,
-			fullscreenButton : false,
-			baseLayerPicker : false,
-			animation : false,
-			shouldAnimate: false,
-			homeButton : false,
-			geocoder : false,
-			navigationHelpButton : false,
-			navigationInstructionsInitiallyVisible : false,
-		};
-		
-		viewerConf.viewerModifier = (viewer: any) => {
-			viewer.screenSpaceEventHandler.removeInputAction(Cesium.ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
-			viewer.bottomContainer.remove();
-		};
-		
-		this.appSettingsService.showTracksLayer = true;
-	}
-	
-	removeAll() {
-		this.tracksLayer.removeAll();
-	}
+export class DemoMapComponent implements AfterViewInit {
+  @ViewChild('layer') tracksLayer: TracksLayerComponent;
+  arcGisMapServerProvider = MapLayerProviderOptions.ArcGisMapServer;
+  sceneMode = SceneMode.SCENE3D;
+  showLayer = true;
+  showMap = true;
+  maps = [
+    {
+      sceneMode: SceneMode.SCENE3D,
+      id: 'main-map',
+      containerId: 'left-map-container',
+    },
+    // {
+    //   sceneMode: SceneMode.PERFORMANCE_SCENE2D,
+    //   id: 'sub-map',
+    //   containerId: 'right-map-container',
+    // },
+  ];
+  mapContainerId = 'left-map-container';
+
+  constructor(
+    private viewerConf: ViewerConfiguration,
+    private appSettingsService: AppSettingsService,
+    private zoomToRect: ZoomToRectangleService,
+  ) {
+    viewerConf.viewerOptions = {
+      selectionIndicator: false,
+      timeline: false,
+      infoBox: false,
+      fullscreenButton: false,
+      baseLayerPicker: false,
+      animation: false,
+      shouldAnimate: false,
+      homeButton: false,
+      geocoder: false,
+      navigationHelpButton: false,
+      navigationInstructionsInitiallyVisible: false,
+      mapMode2D: Cesium.MapMode2D.ROTATE,
+    };
+
+    viewerConf.viewerModifier = (viewer: any) => {
+      viewer.screenSpaceEventHandler.removeInputAction(Cesium.ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
+      viewer.bottomContainer.remove();
+    };
+
+    this.appSettingsService.showTracksLayer = true;
+
+    // setTimeout(() => this.showLayer = false, 5000);
+    // setTimeout(() => this.showMap = false, 5000);
```

Comment:
remove, im not going to be your lint... check where there is unwanted comments

Reply:
For demo purposes